toolchain/check/pattern_match.h

```diff
@@ -30,25 +29,23 @@ struct ParameterBlocks {
 // are matched by the callee, and pattern insts that have a `ParamPattern`
 // as a descendant are matched by the caller.
 //
-// "Calling convention arguments" are the values actually passed from caller to
-// callee at the semantic IR level, and "calling convention parameters" are
-// the corresponding semantic placeholders that they bind to.
+// See EntityWithParamsBase::call_params for a discussion of "call parameters"
```

Comment:
Is this carrying its weight if you fix this to be `Call` parameters? Note too, you had a similar reference below. I could comment about punctuation here, but maybe it should just be removed?

(can't make a suggested edit for this since I'd suggest also removing the blank comment line above)